.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,12 @@
 **What changed? Why?**
 
-
 **Notes to reviewers**
 
-
 **How has it been tested?**
+
+**Does this PR add a new token to the bridge?**
+
```

Comment:
@zencephalon Maybe we should add a first option:

- [ ] No, this PR does not add a new token.

Reply:
Good idea